packages/ui/src/components/editor/panels/Viewport/tools/GridTool.tsx

```diff
@@ -50,10 +50,10 @@ const GridTool = () => {
   }, [])
 
   return (
-    <div className="flex items-center bg-theme-surfaceInput">
+    <div className="flex items-center">
       <Tooltip title={t('editor:toolbar.grid.info-toggleGridVisibility')}>
         <Button
-          startIcon={<MdBorderClear />}
+          startIcon={<MdBorderClear color="#9CA0AA" />}
```

Comment:
`className="text-[#9CA0AA]"` should also work

Reply:
done